docs/ru/query_language/functions/date_time_functions.md

```diff
@@ -43,10 +82,56 @@ SELECT
 Переводит дату-с-временем в число типа UInt8, содержащее номер секунды в минуте (0-59).
 Секунды координации не учитываются.
 
+## toUnixTimestamp
+Переводит дату или дату-с-временем в число типа UInt32 в Unix-дату или Unix-время соответственно. 
+Фактически никакого преобразования значения не происходит, потому что Date это Unix-дата (число дней с 1970-01-01) хранящееся как UInt16. А DateTime это Unix-время (число секунд после 1970-01-01 00:00:00) хранящееся как UInt32.
+
+Пример c типом Date:
+```sql
+SELECT
+    toDate('2019-01-01') AS date,
+    toUnixTimestamp(date) AS UnixTimestamp,
```

Comment:
Ok. BTW, it's surprize for me that `toUnixTimestamp` works for dates in that way.
The number of days from epoch is *not* a unix timestamp.

Maybe you have discovered a bug?
Until we will figure out, we should not include this example in documentation.